resources/js/views/Detail.vue

```diff
@@ -69,16 +69,10 @@
             }
         },
         async mounted() {
-            const { data } = await axios.get(`${process.env.MIX_WEBUMENIA_API}/items/${this.$route.params.id}?locale=${getActiveLanguage()}`)
-            this.item = data.document.content
-        },
-        methods: {
-            getImageSrc(size) {
-                return `${process.env.MIX_WEBUMENIA_URL}/dielo/nahlad/${this.item.id}/${size}`
-            },
-            formatName(name) {
-                return name.replace(/^([^,]*),\s*(.*)$/, '$2 $1')
-            }
+            const response = await axios.get(`/api/items/${this.$route.params.id}`, {headers: {
+                'X-locale': getActiveLanguage()
```

Comment:
Probably won't make a difference, but still:
```suggestion
                'X-Locale': getActiveLanguage()
```